tests/integration/targets/win_dns_record/tasks/tests-DHCID.yml

```diff
@@ -0,0 +1,234 @@
+- name: "TYPE=DHCID - creation (check mode)"
+  win_dns_record:
+    zone: "{{ win_dns_record_zone }}"
+    name: testdhcid
+    value: "AAIBY2/AuCccgoJbsaxcQc9TUapptP69lOjxfNuVAA2kjEA="
+    type: DHCID
+  register: cmd_result
+  check_mode: yes
+
+- name: "TYPE=DHCID - creation get results (check mode)"
+  ansible.windows.win_command: powershell.exe "If (Get-DnsServerResourceRecord -ZoneName '{{ win_dns_record_zone }}' -Name 'testdhcid' -RRType DHCID -Node -ErrorAction:Ignore) { 'exists' } else { 'absent' }"
```

Comment:
Might be easier to use win_powershell here.

Reply:
Probably, I was just keeping it consistent with the rest of the tests.

Still trying to figure out why the creation test isn't working.